enum.go

```diff
@@ -787,14 +768,16 @@ var AllToolCategory = []string{
 type UserRole string
 
 const (
-	UserRoleUser  UserRole = "user"  // A regular user on the account.
-	UserRoleAdmin UserRole = "admin" // An administrator on the account.
+	UserRoleUser      UserRole = "user"       // A regular user on the account.
+	UserRoleAdmin     UserRole = "admin"      // An administrator on the account.
+	UserRoleBasicUser UserRole = "basic_user" // A basic user on the account with limited access.
```

Comment:
This is due to the new RBAC feature flag.  I'm told that if a customer tries to use this when their feature flag is not on they will get an error message saying its not enabled on their account.  So this should be safe